tests/Finder/EnvVariableFinderTest.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+/*
+ * This file is part of the Fidry CPUCounter Config package.
+ *
+ * (c) Théo FIDRY <theo.fidry@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Fidry\CpuCoreCounter\Test\Finder;
+
+use Fidry\CpuCoreCounter\Finder\EnvVariableFinder;
+use PHPUnit\Framework\TestCase;
+use function sprintf;
+
+/**
+ * @covers \Fidry\CpuCoreCounter\Finder\EnvVariableFinder
+ *
+ * @internal
+ */
+final class EnvVariableFinderTest extends TestCase
+{
+    protected function tearDown(): void
+    {
+        putenv('CI_CPU_LIMIT');
+    }
+
+    public function test_it_can_describe_itself(): void
+    {
+        $finder = new EnvVariableFinder('CI_CPU_LIMIT');
```

Comment:
It doesn't directly relate, it's just a generic finder I've made available which tries to get back a positive int from the environment variable. So it can be (and is) used for `KUBERNETES_CPU_LIMIT` or any other.